content/blog/most-used-keyboard-shortcuts.md

```diff
@@ -0,0 +1,90 @@
+---
+author: "Alex Hill"
+date: 2020-01-02
+title: My most used keyboard shortcuts
+---
+
+Anyone who uses a computer for a significant amount of time each day should 
+spend some time learning keyboard shortcuts - they can save you so much time! 
+
+When I started working as a developer, I learnt keyboard shortcuts through pair programming, 
+asking more experienced programmers what their favourite shortcuts were, and just reading
+documentation for the IDE (Intelligent Development Editor) I was using 
+([VisualStudio](https://docs.microsoft.com/en-us/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2019) 
+with [Resharper](https://www.jetbrains.com/help/resharper/Reference__Keyboard_Shortcuts.html) at the time,
+ now [IntelliJ](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts.html).) 
+
+Here are my most used keyboard shortcuts, starting with the basics:
+
+### Global
+These shortcuts are absolute essentials and work in most apps (text editors, browsers, email clients, IDEs, etc.)
+ If you don't already use them, spend some time memorising and practising them. It'll save you loads of time in the long run.
+
+* `ctrl-v` - copy
+* `ctr-x` - cut
+* `ctrl-v` - paste
+* `ctrl-z` - undo
+
+I'm on Linux, but these shortcuts work on Mac and Windows too (on Mac `ctrl` is replaced by `cmd`)
+
+### Ubuntu
+* `alt-tab` - toggle between open applications
+    * plus use the arrow keys to toggle between instances of an app
+    
+<img src="/img/tabchange.gif" alt="Use alt-tab to toggle between apps" />
+
+* `ctrl-shift-v` - paste into the Ubuntu terminal
+* `ctrl-R` - reverse search commands in the Ubuntu terminal. I only learnt this one recently (thanks Rich!) and it's
+been a game changer!
+
+<img src="/img/ctrlR.gif" alt="Reverse search in the terminal" />
+    
+### Browsers
+These work in Chrome, Firefox and IE. Again I use these many many times each day.
+
+* `ctrl-tab` - jump between open tabs
+* `ctrl-t` - open a new tab
+* `ctrl-w` - close the current tab
```

Comment:
How about <kbd>ctrl</kbd>-<kbd>shift</kbd>-<kbd>t</kbd> (re-open last closed tab/window)? I use this one all the time and it seems not that well known

Reply:
I did not know that one! Not technically one of my favourites then, by definition, but sounds like a great one to include... Or we should do a series where we each give our favourites :laughing:

Reply:
In fact I think it'd be fun to do a follow up post where everyone in the team adds 1 or 2 of their most used shortcuts. I think I'll canvas you all and write a follow up post!